cloudify/tests/asserts.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (c) 2017 GigaSpaces Technologies Ltd. All rights reserved
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package tests - fake classes for api testing.
+*/
+package tests
+
+import (
+	"testing"
+)
+
+// AssertEqual - check values
+func AssertEqual(t *testing.T, value, needed interface{}, format string, v ...interface{}) {
```

Comment:
@0lvin  There is a package available for assertion and we do not need to add a new one , I used it before check it https://godoc.org/github.com/stretchr/testify/assert

Reply:
For use it need to put it to vendor. And we can go without additional package if we use only one really simple func from it.

Reply:
And yes of course, when we will use mock logic from testify - we add testify as git submodule to vendor. Without such we broke "make test" on https://github.com/cloudify-incubator/cloudify-kubernetes-provider/blob/master/Makefile#L149

Reply:
Thanks For clarification